Lib/Metaiterators.hpp

```diff
@@ -1656,7 +1775,15 @@ template<class Array> auto arrayIter(Array const& a) { return arrayIter(
 template<class Array> auto arrayIter(Array     && a) { return arrayIter(std::move(a), a.size()); }
 template<class Array> auto arrayIter(Array      & a) { return arrayIter(          a , a.size()); }
 
-}
+template<class Item, class... Items>
+auto iterItems(Item item, Items... items) 
+{ return arrayIter(Stack<Item>{std::move(item), std::move(items)...}); }
+
+template<class Item>
+auto iterItems()
+{ return arrayIter(Stack<Item>{}); }
```

Comment:
The empty iterator?

Reply:
Yes. it's only there so we can use `iterItems<Type>(as...)` with an empty list `as` (as well as a non-empty one ofc).